coverage.sh

```diff
@@ -1,21 +1,35 @@
 #!/bin/bash
+
+WORKSPACE_CRATES=($(cargo metadata --format-version 1 | jq -r '.workspace_members[] | split("/") | .[-1] | split("#")[0]' | sort))
+EXPECTED_WORKSPACE_CRATES=("api" "blob_store" "cancel" "collection" "common" "dataset" "gpu" "io" "issues" "memory" "qdrant" "quantization" "segment" "sparse" "storage")
+
+if [ "${WORKSPACE_CRATES[*]}" != "${EXPECTED_WORKSPACE_CRATES[*]}" ]; then
+    echo "Workspace crates have changed. Please update coverage.sh to update the expected and whitelisted ones."
+    exit 1
+fi
+
+WHITELISTED_WORKSPACE_CRATES=("api" "common" "segment" "blob_store")
```

Comment:
Why whitelist, and not run on everything?

Reply:
I'm running everything now. This looks like an old commit :thinking: